scripts/linters/pre_commit_linter.py

```diff
@@ -494,6 +493,7 @@ def main(args=None):
             third_party_semaphore, name='third_party')
         tasks_third_party.append(task_third_party)
 
+    # Execute tasks.
     # Concurrency limit: 25.
```

Comment:
Can you please explain the rationale why we distinguish custom and third-party linters and why their repective concurrencies are set to what they are?

Please add this rationale as detailed comments in the code. Explain the "why" clearly so that, if someone wants to change these numbers in the future, they know what the considerations are for doing so.

Reply:
Done.